### ISSUE TITLE
Displaying a link to localhost:3000 with npm run dev

### ISSUE THREAD

**Issue by rhit-bowerirn** [sellersindustry/SherpaJS#62]
**Is your feature request related to a problem? Please describe.**
It's annoying when I run npm run dev to have to manually type in the url to localhost:3000

**Describe the solution you'd like**
It would be nice to have the build script print the link like other frameworks

**Comment by SellersEvan** [comment]
This can be implemented by instead of just `console.log` the port we can do this...
```typescript
console.log(`SherpaJS Server is started at "http://localhost:${this.port}".`);
```

**Comment by SellersEvan** [comment]
Perfect! This issue will be shipped with [V1.2.0](https://github.com/sellersindustry/SherpaJS/milestone/2).

Big thanks to our first contributor @rhit-bowerirn 🎉
